server/testutil/testdigest/testdigest.go

```diff
@@ -39,28 +38,58 @@ func (r *randomDataMaker) Read(p []byte) (n int, err error) {
 	}
 }
 
-func NewRandomDigestReader(t testing.TB, sizeBytes int64) (*repb.Digest, io.ReadSeeker) {
-	randomSeedOnce.Do(func() {
-		randomSrc = &randomDataMaker{rand.NewSource(time.Now().Unix())}
-	})
+type Generator struct {
+	randMaker *randomDataMaker
+}
```

Comment:
can we move this to a package that is not in a testing package? I'd like to keep those testonly while also being able to use this generator in other tools (like smash, which I think has a copy of this same code?)

Reply:
Done.